### ISSUE TITLE
Config file issues.max-same is actually issues.max-same-issues

### ISSUE THREAD

**Issue by calind** [golangci/golangci-lint#216]
Docs say that
```yaml
issues:
  max-same: 0
```

Should disable maximum count of issues with the same text, but actually this does it:
```yaml
issues:
  max-same-issues: 0
```
